scripts/pre_redeploy.sh

```diff
@@ -106,8 +106,17 @@ pushd ${LEAPFROG_DIR}
 popd
 
 if [[ ! -f "${UPGRADE_LEAP_MARKER_FOLDER}/configure-apt-sources-rpc.complete" ]]; then
-   pushd ${RPCO_DEFAULT_FOLDER}/rpcd/playbooks
-       openstack-ansible configure-apt-sources.yml
-   popd
-   log "configure-apt-sources-rpc" "ok"
+    pushd ${RPCO_DEFAULT_FOLDER}/rpcd/playbooks
+        openstack-ansible configure-apt-sources.yml
+    popd
+    log "configure-apt-sources-rpc" "ok"
 fi
+
+# Set openstack_domain to empty which makes fqdns hostname correct
+pushd /etc/openstack_deploy/
+    if ! grep "openstack_domain" user_osa_variables_defaults.yml > /dev/null; then
+        echo "openstack_domain: ''" >> user_osa_variables_defaults.yml
+    else
+        sed -i s/^openstack_domain.*/openstack_domain:\ \'\'/g user_osa_variables_defaults.yml
```

Comment:
I would always single quote the sed script
Perhaps you can just simplify it via:

```
sed -i '/^openstack_domain.*/d' user_osa_variables_defaults.yml
echo "openstack_domain: ''" >> user_osa_variables_defaults.yml
```

As you seem to overwrite any opensack_domain settings regardless if existing or not